adapters/bidder.go

```diff
@@ -39,6 +39,20 @@ type Bidder interface {
 	MakeBids(internalRequest *openrtb.BidRequest, externalRequest *RequestData, response *ResponseData) (*BidderResponse, []error)
 }
 
+// TimeoutBidder is used to identify bidders that support timeout notifications.
+type TimeoutBidder interface {
+	MakeRequests(request *openrtb.BidRequest, reqInfo *ExtraRequestInfo) ([]*RequestData, []error)
```

Comment:
Why are the standard bidder methods duplicated here?

Reply:
Not 100% sure it is needed, but it does make clear that it is a bidder rather than some other random thing, and can still be used for bidder things.